pkg/tracker/newCurrentVariables.go

```diff
@@ -53,7 +55,7 @@ func (b *NewCurrentVariablesTracker) Exec(ctx context.Context) error {
 	instanceGetter := ctx.Value(tellorCommon.ContractsGetterContextKey).(*getter.TellorGetters)
 	myStatus, err := instanceGetter.DidMine(nil, returnNewVariables.Challenge, fromAddress)
 	if err != nil {
-		fmt.Println("My Status Retrieval Error")
+		level.Error(logger).Log("msg", "My Status Retrieval Error", "err", err)
```

Comment:
For all errors, there is a golang practice to use all lower case and I like to do the same for the logging. Also, I don't put words like `error` in the message as it gets repeated twice in the final output.
```suggestion
		level.Error(logger).Log("msg", "my status retrieval", "err", err)
```